src/multivariate/dirichlet.jl

```diff
@@ -118,7 +118,7 @@ function dirichlet_mode!(r::AbstractVector{<:Real}, α::AbstractVector{<:Real},
 end
 
 function dirichlet_mode(α::AbstractVector{<:Real}, α0::Real)
-    all(αi < 1 for αi in α) || error("Dirichlet has a mode only when alpha[i] > 1 for all i")
+    all(αi > 1 for αi in α) || error("Dirichlet has a mode only when alpha[i] > 1 for all i")
```

Comment:
Thanks, yes this seems to be a typo! A bit faster for `FillArray`s would be
```suggestion
    all(x -> x > 1, α) || error("Dirichlet has a mode only when alpha[i] > 1 for all i")
```
So maybe let's change it?